Makefile

```diff
@@ -35,3 +35,7 @@ clean-data:
 .PHONY: describe-data
 describe-data:
 	python -m ${BASEDIR} --describe-data
+
+.PHONY: logistic-regress
+logistic-regress:
```

Comment:
I think it'd be better to have it just be `logistic-regression` as the other grammatically correct way would be to put it as `regress-logistically`, I believe.

Reply:
sounds good. can change names!